wiz-admission-controller/values.yaml

```diff
@@ -463,3 +463,19 @@ global:
   affinity: {} # custom affinity rules for node assignment
   tolerations: [] # custom tolerations
   clusterExternalId: "" # Required for OKE clusters - specify the cluster's OCID
+
+  # Wiz Helm Charts process the security contexts
+  # in the following order:
+  # 1. Pod security context
+  #    overridePodSecurityContext -> podSecurityContext
+  # 2. Containers security context
+  #    overrideSecurityContext -> securityContext
+  #
+  # Adding the following overrides to your user-supplied values.yaml
+  # allows you to customize and override the defaults
+  # (useful for e.g. OpenShift installations that automatically
+  #  mutate the Pod and use the first UID and first GID from
+  #  the allowed range to run the Pod).
+  #
+  # overridePodSecurityContext: {}
```

Comment:
1. Put it under podSecurityContext value.
2. We don't support PodSecurityContext under global because sensor/ ac/ broker need a different context
3. Same for SecurityContext
4. Same for the broker